src/components/Profile/Item.tsx

```diff
@@ -2,16 +2,40 @@ import React from 'react';
 
 export default function Item(props) {
   return (
-    <div className="profile-item" onClick={() => props.goToChat(0)}>
+    <div className="profile-item" onClick={() => props.goToChat(props.index)}>
       {props.checkmark ? (
-        <div className="profile-item-checkmark" />
+        <div className="profile-item-checkmark">
+          <img
+            src="icons/check-mark-3-512.png"
+            alt="checkmark"
+            style={{
+              width: '30px',
+              display: 'table-cell',
+              margin: 'auto',
+              marginTop: '10px',
+            }}
+          />
+        </div>
       ) : (
-        <div className="profile-item-emptycheckmark" />
+        <div className="profile-item-emptycheckmark">
+          <img
+            src="icons/check-mark-3-512.png"
+            alt="checkmark"
+            style={{
```

Comment:
I think this img tag qualifies to have its own class in the css instead of inline styling. (To keep things consistent)